src/main/index.ts

```diff
@@ -256,6 +256,12 @@ app.on("ready", async () => {
     LensProtocolRouterMain.getInstance().rendererLoaded = true;
   });
 
+  ipcMainOn(IpcMainWindowEvents.OPEN_CONTEXT_MENU, (event) => {
```

Comment:
Should this be within `initializers.initIpcMainHandlers()` ?

Reply:
Good catch, moved this into initializers.